inform-db/src/main/java/uk/ac/ucl/rits/inform/informdb/forms/FormAnswer.java

```diff
@@ -0,0 +1,111 @@
+package uk.ac.ucl.rits.inform.informdb.forms;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import uk.ac.ucl.rits.inform.informdb.TemporalCore;
+import uk.ac.ucl.rits.inform.informdb.annotation.AuditTable;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import java.time.Instant;
+
+/**
+ * Stores the value assigned to a particular instance of an answered form question.
+ * Eg. the value of a Smart Data Element (SDE).
+ */
+@Entity
+@Data
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+@AuditTable
+public class FormAnswer extends TemporalCore<FormAnswer, FormAnswerAudit> {
+    /**
+     * \brief Unique identifier in EMAP for this instance of an SDE.
+     */
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long formAnswerId;
+
+    /**
+     * \brief Metadata for this answer - ie. what was the question?
+     */
+    @ManyToOne
+    @JoinColumn(name = "formQuestionId")
+    private FormQuestion formQuestionId;
+
+    /**
+     * \brief The instance of a filled-in form that this answer belongs to.
+     */
+    @ManyToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "formId")
+    private Form formId;
+
+    /**
+     * \brief A unique ID for this form answer that can be used to track back to the source system.
+     * For an Epic SDE this would be the HLV ID (HLV .1)
+     */
+    private String formAnswerSourceId;
+
+    /**
+     * \brief Current value of the SDE - may be a multi-line string concatenated together.
+     * HLV 50.
+     */
+    @Column(columnDefinition = "text")
+    private String valueAsString;
```

Comment:
To match what we've previously used in other data types
```suggestion
    private String valueAsText;
```

Reply:
This doesn't look like it's changed, and is a blocking change